docs/releases/1.22-NOTES.md

```diff
@@ -34,6 +34,10 @@ Currently this is only available using the AWS cloud provider.
 
 kOps can now provision instances with nvidia GPUs and configure it for container workloads without the need of hooks and operators. See [GPU support](https://kops.sigs.k8s.io/gpu/)
 
+## Breacking change in NodeLocalDNS
+
+Since 1.22.0 Cluster `spec.kubeDNS.nodeLocalDNS.forwardToKubeDNS` default behaviour changes from `true` to `false`.
```

Comment:
TIL, it probably would have been smarter to change the default behavior from 'false' to 'true' when fixing the template (https://github.com/kubernetes/kops/pull/11743).

Reply:
That will generate another BC change. At current stage, documentation is enough IMO.

Reply:
I think for the majority of people, false is the way to go. I'm ok with the false as default. Just need to be documented in the release.

Reply:
False is definitely the way to go. I think very few do custom core files.